_posts/blog/2018-10-10/2018-10-10-how-to-buy-cryptocurrencies-privately-with-no-fees.markdown

```diff
@@ -0,0 +1,86 @@
+
+Then choose payment type. Unless you fully trust the vendor, you should use a moderator payment with a verified moderator. Enter the amount you wish to buy, then review your order and click “Pay.” You’ll now see the payment screen.
+<br>  
+![Buy the Cryptocurrency](examplepurchase.gif "Buy the Cryptocurrency")
+<br>  
+## Step #6: Send Payment
+
+Once you’ve placed the order you now need to pay for it. There are two ways to pay in OpenBazaar. One is by scanning the QR code on a mobile wallet or copying the details at the payment screen into a separate wallet and sending payment. The other method is using the internal OpenBazaar wallet.
+
+If you don’t want to use a separate wallet, or you try using an external wallet and you receive an error message, then it’s best to use the internal OpenBazaar wallet and click the “Pay from Wallet” button. 
+
+Before you can pay from the OpenBazaar wallet, you’ll need to fund it. To do this, click on the wallet button, then in the wallet interface click “Receive Money.” This will display a QR code and address you can use to fund the wallet. Note that you’ll need to wait for one confirmation on the blockchain before the funds are spendable.
+<br>  
+![Receive funds in OpenBazaar Wallet](obwalletreceive.gif "Receive funds in OpenBazaar Wallet")
+<br>  
+
+To pay for orders you’ve already placed, click on “Purchases” in the dropdown menu on the top right (your profile picture) and then click on the specific order.
+
+Once payment has been sent you’ll see a green check mark appear and it’s now up to the vendor to fulfill your order.
+<br>  
+## Step #7: Check your order
+
+To check your order, click on “Purchases” in the dropdown menu on the top right (your profile picture) and then click on the specific order. You’ll see a bar showing the progress of the order. 
+
+Once the seller fulfills the order you should see it showing “fulfilled,” and the vendor should send confirmation such as a transaction hash to prove they sent the funds to the address you provided. Enter the transaction hash into a block explorer to verify, or just check the wallet where you expected the funds to be sent. If you’ve received them, leave a review and close the order out. It’s important that you do finish the order, otherwise the funds will not be released from escrow and the seller will be forced to open a dispute to claim their funds.
+
+If you don’t receive the funds in a reasonable period, message the seller to ask why. If you get no response after a while or something else is wrong with your order, you can choose to open a dispute with the moderator by clicking the “Dispute Order” button when viewing the order in your purchases.
+
+OpenBazaar is also a platform for goods and services as well. While you’re waiting for the vendor to send your new coin, check out what the marketplace has to offer.
```

Comment:
"is also" and "as well" are functionally the same, you could remove one of them.